chains/orchestrator-relays/runtime/dancelight/src/xcm_config.rs

```diff
@@ -268,7 +268,7 @@ impl pallet_xcm::Config for Runtime {
     type ExecuteXcmOrigin = xcm_builder::EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
     type XcmExecuteFilter = Everything;
     type XcmExecutor = XcmExecutor<XcmConfig>;
-    type XcmTeleportFilter = Everything;
+    type XcmTeleportFilter = Nothing;
```

Comment:
This should change for benchmarks only? Or do we want to change it for real runtime as well?

Reply:
for everything. We should not allow teleports